crates/cast/src/cmd/wallet/mod.rs

```diff
@@ -454,7 +454,9 @@ impl WalletSubcommands {
                 } else {
                     wallet.sign_message(&Self::hex_str_to_bytes(&message)?).await?
                 };
-                sh_println!("0x{}", hex::encode(sig.as_bytes()))?;
+                sh_println!("message: {}", message)?;
+                sh_println!("address: {}", wallet.address())?;
+                sh_println!("signature: 0x{}", hex::encode(sig.as_bytes()))?;
```

Comment:
We have the `shell::is_json();`  check to output JSON compatible data

If `shell::verbosity() > 0` we should:

When in `--json` mode, print a single JSON formatted version of 

```json
{
  "message": ..,
  "address": ..,
  "signature": ...
}
```

When we are not in `--json` mode we should render this as a nicely formatted message optimized for readability, preferably using a single `sh_println!`

If we are not in verbosity mode we should only render what we had before:

```rust
sh_println!("0x{}", hex::encode(sig.as_bytes()))?;
```

Reply:
Thanks this makes sense